deploy/helm/kuberhealthy/values.yaml

```diff
@@ -106,6 +106,16 @@ check:
       tag: v1.2.2
     extraEnvs:
     nodeSelector: {}
+  networkConnection:
```

Comment:
We don't have to add every new check to the helm chart here - we can simply add them to the check registry.

Actually, I don't even mind if the check is sourced from another git repository and image host entirely!  This means you could self-publish this check and then open a small PR to update the `EXTERNAL_CHECKS_REGISTRY.md` file.